### ISSUE TITLE
Keyboard accessibility of personal settings Sessions menu

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#12243]
This is about hte sessions/devices menu in the personal settings:

![sessions now](https://user-images.githubusercontent.com/925062/47853024-c5bfea80-dddd-11e8-93b9-eb6a9e30c73c.png)

The 3-dot menu needs to open on enter, and the Revoke button needs to work on enter too. (Filesystem access toggling works via Space, as is normal). I tried this in `settings/js/authtoken_view.js` but it does not work. cc @nextcloud/javascript 

```javascript
            $el.on('click', 'a.icon-delete', _.bind(this._onDeleteToken, this));
            this._deviceRevoke = $('a.icon-delete');
            this._deviceRevoke.on('keypress', function(event = key) {
                if (key === 13) {
                    console.log('Enter delete');
                    this._onDeleteToken(this);
                }
            }.bind(this));
 
            $el.on('click', '.icon-more', _.bind(this._onConfigureToken, this));
            this._deviceSettings = $('.icon-more');
            this._deviceSettings.on('keypress', function(event) {
                if (event === 13) {
                    console.log('Enter settings');
                    this._onConfigureToken(this);
                }
            }.bind(this));
```

**Comment by MorrisJobke** [comment]
Moved to 16.

**Comment by jancborchardt** [comment]
This was fixed some time ago by moving to the ActionMenu component.
